src/index.html

```diff
@@ -13,6 +13,52 @@
     />
```

Comment:
The closing bracket of the <link> tag should be aligned with the opening tag. Ensure consistent formatting for better readability.